util/MySqlMigrations/HelperScripts/2024-03-22_00_EnableOrgsCollectionEnhancements.sql

```diff
@@ -33,113 +33,107 @@
             `OU`.`Permissions` IS NOT NULL AND
             JSON_VALID(`OU`.`Permissions`) AND JSON_VALUE(`OU`.`Permissions`, '$.editAssignedCollections') = 'true'));
 
--- Start transaction
-START TRANSACTION;
```

Comment:
Yep, this is my understanding too, you can check the SQL it will generate [with the CLI](https://learn.microsoft.com/en-us/ef/core/cli/dotnet#dotnet-ef-migrations-script) and see that it will add transactions. Also, when you don't add them yourself and let EF do it then it will respect the `--no-transactions` option on that command. Why you'd want to do that I have no idea but it's there 🤷.

Reply:
Thanks both of you for the clarification.